mantidproject/mantid
# Imaging GUI: Merge the Run and Setup tabs

- [x] Requres #18258 merged first

---

## Description of work.

- The tabs that were previously `Run` and `Setup` separately, have now been merged into a single one, just called `Run`.
   - Any suggestions to further improve/change the merge are welcome.
- The tab ordering of the tab stops in the `Run` tab has been improved.

## To test:

- Test functionality from both Run and Setup tabs 
  - Selecting a different tool and path works when running a reconstruction
- Check that the ordering when pressing Tab is sensible (it used to jump from password to reconstruction tool to username, etc., which is bad for navigation)



Fixes #18134 .



[*Added in imaging.rst file*: 5180d1](https://github.com/mantidproject/mantid/commit/5180d1453c56a1f8717b3adaf9ef03a9dc547223)

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
